index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contatos</title>
+</head>
+<body>
+    <h1>Deixe aqui sua sugestão de melhoria</h1>
+
+    <form action="mailto:estephanie.maga@gmail.com">
+             <label for="Nome">Nome:</label>
+             <input type="text" name="Nome" id="Nome" required>            
+             <label for="Sobrenome">Sobrenome:</label>
+             <input type="text" name="Sobrenome" id="Sobrenome" required>
+             <br>
+             <label for="E-mail">E-mail</label>
+             <input placeholder="ex: abc@gmail.com" name="E-mail" id="E-mail" required>
```

Comment:
Poderia usar o type email nesse caso, o navegador consegue fazer uma validação pra esses tipos nativamente
```suggestion
             <input type="email" placeholder="ex: abc@gmail.com" name="E-mail" id="E-mail" required>
```